oak_functions/loader/src/grpc.rs

```diff
@@ -129,7 +140,20 @@ struct ErrorLogger {
     logger: Logger,
 }
 
-impl LogError for ErrorLogger {
+impl grpc_streaming_attestation::server::LogError for ErrorLogger {
+    fn log_error(&self, error: &str) {
+        self.logger.log_public(
+            Level::Warn,
+            &format!(
+                "{:?}: Remote attestation error: {}",
+                std::thread::current().id(),
+                error
+            ),
+        );
+    }
+}
+
+impl grpc_unary_attestation::server::LogError for ErrorLogger {
```

Comment:
Duplicated this trait for pragmatism. We could probably use the same trait for the two, but not sure if that's needed, given that eventually we'll support unary requests.

ref the previous discussion on unary & streaming servers implementing the same interface: https://github.com/project-oak/oak/pull/2584#pullrequestreview-900587814

Reply:
How about moving that trait in oak_utils or some other common dependency of both? It seems confusing to have two traits with identical names and functionality (unless there is no easy way around it).

Reply:
I don't think oak_utils is an appropriate place for the dependency, as it is mostly used as a dev dependency to compile protos. I think a new create (perhaps oak_functions_common or something like that) would be a good idea. It can then be reused for the extension-based-traits mentioned in #2576

Reply:
Fair enough, it sounds like such a crate may be useful at some point. For now feel free to go ahead with whatever works @jul-sh